Services/DataX.Gateway/DataX.Gateway.Api/Controllers/GatewayController.cs

```diff
@@ -153,7 +166,7 @@ public async Task<ApiResult> QueryService(HttpRequestMessage request, string app
         private Dictionary<string, string> FetchUserHeaders()
         {
             var headers = new Dictionary<string, string>();
-            var userName = ClaimsPrincipal.Current.FindFirst(ClaimTypes.Upn) != null ? ClaimsPrincipal.Current.FindFirst(ClaimTypes.Upn).Value : (ClaimsPrincipal.Current.FindFirst(ClaimTypes.Email)?.Value);
+            var userName = ClaimsPrincipal.Current.FindFirst(ClaimTypes.Upn) != null ? ClaimsPrincipal.Current.FindFirst(ClaimTypes.Upn).Value : ClaimsPrincipal.Current.FindFirst(ClaimTypes.Email)?.Value;
```

Comment:
ClaimTypes.NameIdentifier

Reply:
"http://schemas.microsoft.com/identity/claims/objectidentifier" is actually ClaimTypes.NameIdentifier

Reply:
yes. Updated the code